plugin/src/main/java/org/opennms/velocloud/requisition/AbstractRequisitionProvider.java

```diff
@@ -48,6 +48,10 @@
 
     private final VelocloudApiClientProvider clientProvider;
 
+    public static final String VELOCLOUD_METADATA_CONTEXT = "velocloud";
+
+    public static final String VELOCLOUD_LINK_SERVICE_NAME = "velocloud_link";
```

Comment:
```suggestion
    public static final String VELOCLOUD_LINK_SERVICE_NAME = "VelocloudLink";
```

to follow common service naming.